src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -38,15 +42,39 @@ def __init__(self):
     for extension in extensions_to_strip:
       app_name = utils.strip_from_right(app_name, extension)
 
-    self.trials = list(
+    db_trials = list(
         data_types.Trial.query(data_types.Trial.app_name == app_name))
 
+    self.trials = OrderedDict()
```

Comment:
Is this just to ensure the insertion order is kept? In that case, a [plain dict in Python 3](https://docs.python.org/3/library/collections.html#ordereddict-objects) will suffice.

Reply:
done, changed to regular dict